docker/base-deps/Dockerfile

```diff
@@ -6,14 +6,9 @@ RUN apt-get update \
         git \
         wget \
         cmake \
-        pkg-config \
```

Comment:
can we get rid of `cmake` above?

Reply:
I think I tried and it was needed to pip install gym